io/plugins/eu.esdihumboldt.hale.io.gml/src/eu/esdihumboldt/hale/io/gml/writer/internal/StreamGmlWriter.java

```diff
@@ -149,11 +151,17 @@
 	public static final String PARAM_OMIT_NIL_REASON = "xml.notNil.omitNilReason";
 
 	/**
-	 * The parameter name for the flag specifying if the output of geometry
+	 * The name of the parameter specifying how the output of geometry
 	 * coordinates should be formatted.
 	 */
 	public static final String PARAM_GEOMETRY_FORMAT = "geometry.write.decimalFormat";
 
+	/**
+	 * The name of the parameter specifying how the output of Double values
+	 * should be formatted.
+	 */
+	public static final String PARAM_DECIMAL_FORMAT = "gml.decimalFormat";
```

Comment:
I would prefer `xml.` as prefix as this is not specific to GML but applies to any type of XML output.